spec/struct.dd

```diff
@@ -831,6 +831,75 @@ void main()
         qualified with the more restrictive qualifier, which is `immutable`.
         )
 
+        $(P The postblits `__fieldPostblit` and `__aggrPostblit`
+        are generated without any implicit qualifiers and are not considered
+        struct members. This leads to the situation where qualifying an
+        entire struct declaration does not have any impact on the above
+        mentioned postblits. However, since `__xpostblit` is a member of
+        the struct and an alias of one of the other postblits, the qualifiers
+        applied to the struct will affect the aliased postblit:)
+
+---
+struct S
+{
+    this(this)
+    { }
+}
+
+// `__xpostblit` aliases the aggregated postblit so the `const` applies to it.
+// However, the aggregated postblit calls the field postblit which does not have
+// any qualifier applied, resulting in a qualifier mismatch error
+const struct B
+{
+    S a;        // error : mutable method B.__fieldPostblit is not callable using a const object
+    this(this)
+    { }
+}
+
+// `__xpostblit` aliases the field postblit; no error
+const struct B2
+{
+    S a;
+}
```

Comment:
Maybe worth splitting up into multiple tests (?), s.t. you can use `SPEC_RUNNABLE_EXAMPLE_COMPILE`